trace-mini-agent/src/trace_processor.rs

```diff
@@ -95,7 +111,10 @@ impl TraceProcessor for ServerlessTraceProcessor {
                 trace_utils::compute_top_level_span(&mut chunk.spans);
             }
 
-            trace_utils::set_serverless_root_span_tags(&mut chunk.spans[root_span_index]);
+            trace_utils::set_serverless_root_span_tags(
+                &mut chunk.spans[root_span_index],
+                config.gcp_function_name.clone(),
```

Comment:
what about passing the ref here to avoid a clone (as you're doing a toString in set_serverless_root_span_tags)?

Reply:
i think we should keep it as is, because `set_serverless_root_span_tags` doesn't re-clone `gcp_function_name`. 

Making this parameter a &str would require using.to_string() in `set_serverless_root_span_tags` anyways as the span's Meta map has String values, not &str.

Reply:
oh yes, I got confused with other to_string() but they're actually not on the gcp_function_name param